### ISSUE TITLE
[NewsList] Cannot manage newslist target if not in content-management but admin

### ISSUE THREAD

**Issue by srenault-meeds** [Meeds-io/meeds#2459]
**Steps to reproduce**

1. As platform admin, but not content-manager (no permission in group web-contributors)
2. Access the newslist target management
3. Edit a target
4. Add permissions
5. Save
6. See error: it doesn't work because the admin is not in the group web-contributors
![Edit target of newslist](https://github.com/user-attachments/assets/df3fb11f-5ffc-4410-83e6-607d3b6dbeb5)

**Expected behavior**

Consider platform admin (* in platform/admin) as the super user so he can manage targets as expected

**Comment by srenault-meeds** [comment]
FYI @Julien-Dubois-eXo

**Comment by Julien-Dubois-eXo** [comment]
@srenault-meeds I agree, it's better to use platform/admin to centralize all admin permissions instead of several groups. This was done on purpose, as before, all the features related to content management were web-contributor. So maybe it's more an improvement rather than a bug.

**Comment by srenault-meeds** [comment]
ok.
Do you plan to improve it?
Thanks

**Comment by Julien-Dubois-eXo** [comment]
From what I know it's not part of the scope now. I asked @margondicco if I should include this point and the other about target permission by default on the MIP161.

**Comment by srenault-meeds** [comment]
@Julien-Dubois-eXo Can you please inform me about this one?

**Comment by Julien-Dubois-eXo** [comment]
It's planned but no longer treated. It will be done during February probably.
